util/src/test/java/io/grpc/util/OutlierDetectionLoadBalancerTest.java

```diff
@@ -512,7 +512,7 @@ public void successRateOneOutlier_configChange() {
 
     loadBalancer.acceptResolvedAddresses(buildResolvedAddress(config, servers));
 
-    generateLoad(ImmutableMap.of(subchannel2, Status.DEADLINE_EXCEEDED), 8);
```

Comment:
This means that the helper is asked to update the balancing state four more times than before - doesn't that indicate a problem somewhere? Or is there an understood reason why the updates increase?

Reply:
It is because there is another layer.  This is the PickFirstLBs doing their updates between the subchannel and petiole policy.